src/azure-cli/azure/cli/command_modules/acr/_errors.py

```diff
@@ -104,6 +104,12 @@ def format_error_message(self, *args):
 )
 
 
+CONNECTIVITY_SSL_ERROR = ErrorClass(
+    "CONNECTIVITY_SSL_ERROR",
+    "Unable to establish a secure connection to the registry '{}'. Are you using a proxy server?"
```

Comment:
Maybe `... to the endpoint '{}'`? Considering that you are using `login_server` to handle the error.

Reply:
I wanted to add an endpoint but wanted to keep it consistent with other Error messages.

For instance,  CONNECTIVITY_FORBIDDEN_ERROR also specifies registry and passes login_server. 
https://github.com/AzureCR/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acr/_errors.py#L79

https://github.com/AzureCR/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acr/check_health.py#L204

Reply:
It looks like for all CONNECTIVITY related errors, `registry {}` is being used.  For FORBIDDEN_ERROR too registry {} is used and login_server is passed as an argument.

@ni-reis

Reply:
Sounds good then, let's keep consistency. In case this causes confusion we can change all of them to "endpoint"